planning/behavior_path_planner/docs/behavior_path_planner_drivable_area_design.md

```diff
@@ -104,22 +107,53 @@ Note that we only expand right bound of the rightmost lane and left bound of the
 
 #### Dynamic Expansion
 
-The drivable area can also be expanded dynamically by considering the ego vehicle footprint projected on each path point.
-This expansion can be summarized with the following steps:
+The drivable area can also be expanded dynamically based on a minimum width calculated from the path curvature and the ego vehicle's properties.
+
+| Without dynamic expansion                                                  | With dynamic expansion                                                   |
+| -------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| ![dynamic_expansion_off](../image/drivable_area/dynamic_expansion_off.png) | ![dynamic_expansion_on](../image/drivable_area/dynamic_expansion_on.png) |
+
+Next we detail the algorithm used to expand the drivable area bounds.
+
+##### 1 Calculate and smooth the path curvature
+
+To avoid sudden changes of the expanded drivable area, we first try to reuse as much of the previous path and its calculated curvatures as possible.
```

Comment:
Is the `expanded drivable area` here refers to `static` or `dynamic`?

Reply:
I made it more clear that it was for the dynamic expansion (fd11526ab9).